public/video-ui/src/components/VideoUpload/AddSelfHostedAsset.js

```diff
@@ -27,7 +27,7 @@ export default class PlutoProjectPicker extends React.Component {
       <div className="video__detailbox video__detailbox__assets">
         <div className="video__detailbox__header__container">
           <header className="video__detailbox__header">
-            Guardian Hosted Video
+            Self-Hosted Video
```

Comment:
https://github.com/guardian/media-atom-maker/pull/641 allowed Hosted to upload to YouTube, rename the form label for explicitness